lua/nvim-web-devicons/icons-default.lua

```diff
@@ -155,6 +155,18 @@ local icons_by_filename = {
     cterm_color = "113",
     name = "Zshrc",
   },
+  [".zsh_plugins.txt"] = {
+    icon = "󱉋",
+    color = "#ff0288",
+    cterm_color = "198",
+    name = "Antidote",
+  },
```

Comment:
```suggestion
```

As per https://github.com/nvim-tree/nvim-web-devicons/pull/422#discussion_r1531557602, this isn't stardard zsh file. Please remove.

Reply:
ok